SPECS/php-pecl-apcu/php-pecl-apcu.spec

```diff
@@ -0,0 +1,456 @@
+# Fedora spec file for php-pecl-apcu
+#
+# Copyright (c) 2013-2024 Remi Collet
+# License: CC-BY-SA-4.0
+# http://creativecommons.org/licenses/by-sa/4.0/
+#
+# Please, preserve the changelog entries
+#
+
+%global pecl_name  apcu
+%global with_zts   0%{?__ztsphp:1}
+%global ini_name   40-%{pecl_name}.ini
+%global sources    %{pecl_name}-%{version}
+
+Name:           php-pecl-apcu
```

Comment:
Triggered a test build on behalf of @sallhms :
- [PR-10414](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=638641&view=results)